jobtracker.go

```diff
@@ -77,8 +77,11 @@ func getJSON(url string, data interface{}) error {
 	if err != nil {
 		return err
 	}
-
-	return json.Unmarshal(jsonBytes, data)
+	err = json.Unmarshal(jsonBytes, data)
```

Comment:
does this method return either error or the unmarshal-ed response?

Reply:
great question! it does not. it stores stores the in the value pointed to by `data`.

Reply:
can we then log in the caller of this function then? since we already return it.

Reply:
there are six callers, it seems more efficient and future-proof to log here, no?

Reply:
👍  sounds good